extensions/vscode/src/views/homeView.ts

```diff
@@ -979,6 +984,53 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     return commands.executeCommand(Commands.Logs.Focus);
   }
 
+  private async showAssociateGUID() {
```

Comment:
Is there a good way for us to break this out of `homeView`? The file is getting rather cumbersome, and this feels like a candidate that we could extract into a single file.

Reply:
I've created an `actions` folder, and have moved this method into its own file. The only thing that it requires is that the `PublisherState` be passed into it as a parameter.

If you like this approach, I can see that we could easily move all of the "on" actions that are dispatched from `onConduitMessage`. I think we could go further, but in order to limit the testing impact of a refactoring, I think we should come back after the pending release.